pyanaconda/ui/gui/spokes/advanced_storage.py

```diff
@@ -494,13 +497,58 @@ def get_selected_namespaces(self):
         return namespaces
 
 
+class NVMeFCPage(FilterPage):
+    # Match these to nvmefcTypeCombo ids in glade
+    SEARCH_TYPE_NAMESPACE = 'Namespace'
+    SEARCH_TYPE_CONTROLLER = 'Controller'
+    SEARCH_TYPE_SUBSYSTEMNQN = 'Subsystem NQN'
+
+    def __init__(self, builder):
+        super().__init__(builder, "nvmefcModel", "nvmefcTypeCombo")
+        self._namespace_entry = self._builder.get_object("nvmefcNamespaceEntry")
+        self._controller_entry = self._builder.get_object("nvmefcControllerEntry")
+        self._subsystemnqn_entry = self._builder.get_object("nvmefcSubsystemNqnEntry")
+
+    def is_member(self, device_type):
+        return device_type == "nvme-fabrics"
```

Comment:
The condition would also be true for NVMe/TCP devices, which should not be displayed as NVMe/FC devices.
Would it make sense to have one generic class (NVMeoFPage?) for all NVMe-over-fabric devices? In the future we want to add a support for NVMe/TCP, so NVMe/FC and NVMe/TCP devices could possibly be displayed together at the same page.

Reply:
If it's possible to handle together all "remote" NVMes, that would be good. This might be a terminology issue. I thought Fibre Channel and Fabric is the same thing?

Reply:
Just cleared it up with Vojta - for the UI it should be called "NVMe (over) Fabrics".